docs/integrations/destinations/databricks.md

```diff
@@ -344,7 +344,8 @@ Delta Lake tables are created. You may want to consult the tutorial on
 ## CHANGELOG
```

Comment:
There will be a lot more docs to update once we are ready to release - fine to skip that for now